CMakeLists.txt

```diff
@@ -7,7 +7,11 @@ add_subdirectory(lib/open-simulation-interface)
 get_directory_property(OSI_VERSION_MAJOR DIRECTORY lib/open-simulation-interface DEFINITION VERSION_MAJOR)
 get_directory_property(OSI_VERSION_MINOR DIRECTORY lib/open-simulation-interface DEFINITION VERSION_MINOR)
 get_directory_property(OSI_VERSION_PATCH DIRECTORY lib/open-simulation-interface DEFINITION VERSION_PATCH)
-set(OSIVERSION "${OSI_VERSION_MAJOR}.${OSI_VERSION_MINOR}.${OSI_VERSION_PATCH}")
+# set(OSIVERSION "${OSI_VERSION_MAJOR}.${OSI_VERSION_MINOR}.${OSI_VERSION_PATCH}")
+# CMOSI bugfix, CMOSI interface expects OSI 3.0.0, although supporting 3.5.0
+set(OSIVERSION "3.0.0")
```

Comment:
Since this is only a fix for CarMaker, I would leave the correct OSI version here in the template instead of hardcoding 3.0.0. This is rather a bug in CarMaker than a bug in the template.

Reply:
Yes, I would definitely change it back, I just needed it as a bugfix to test the public logging. I guess it makes sense to do it when merging.

Reply:
I removed the hardcoded values for merging.